db/main/structure.sql

```diff
@@ -2878,7 +2878,8 @@ CREATE TABLE public.users (
     redacted_at timestamp without time zone,
     preferences jsonb DEFAULT '{}'::jsonb,
     vcs_type character varying DEFAULT 'GithubUser'::character varying,
-    vcs_id character varying
+    vcs_id character varying,
+    utm_params character varying
```

Comment:
are you going to store all UTM params in a single column as a string?

Reply:
Yes, this was the format that we've agreed on with @shairyar while discussing the sync with SF. Although, I'm totally open for opinions and discussions here.